src/gui/Popups.java

```diff
@@ -165,4 +118,16 @@ protected static int getDelayOption(int currentDelay, boolean error) {
 			return currentDelay;
 		}
 	}
+	
+	public static boolean getPassConfirmation(ManagedGame game) {
+		if (!game.didPass(game.playerTurn().next())) {
+			// if the previous player didn't pass, passing now has no major consequences
+			return true;
+		}
+		
+		JLabel text = new JLabel("Če zdaj izpustite potezo, se bo igra zaradi dveh zaporednih izpuščenih potez končala.");
```

Comment:
Kje se to pokaže?

Reply:
Če na katerikoli način izpustič potezo, ko je neposredno pred tem nasprotnik izpustil potezo.

Reply:
Na desni klik meni ne deluje